demos/src/json-config.mustache

```diff
@@ -0,0 +1,13 @@
+
+<script type="application/json" data-o-tracking-config>
+    {
+        "server": "https://test.spoor-api.ft.com/px.gif",
+        "context": {
+            "product": "origami" // e.g. webapp, next - This is a defined list, send the correct value!
+        }
+    }
+</script>
+
+<button onclick="var event = new CustomEvent('oTracking.page', { url: document.URL }); window.dispatchEvent(event);">Track a page</button>
```

Comment:
could you make that document.body? I think o-tracking should attach to the body. And if it should still attach to window, it's better if a demo doesn't recommend products to dispatch it on window